src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/RuntimeImports.Wasm.cs

```diff
@@ -0,0 +1,214 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Runtime imports specific to WASM. Currently these are just SPGCT redefinitions
```

Comment:
What does "SPGCT" stand for in this context?

Reply:
**S**u**p**ress**GCT**ransition[Attribute]. I have reworded the comment.